### ISSUE TITLE
Update pako dependency to version ^2.0.2 for better tree shaking

### ISSUE THREAD

**Issue by andreaslarssen** [cantoo-scribe/pdf-lib#24]
### Describe your idea

From https://github.com/Hopding/pdf-lib/issues/689:

> Angular11 generates warnings for pdf-lib because of the old module format of pako
>
> Warning: C:\Users\*****\node_modules\pdf-lib\es\core\PDFContext.js depends on 'pako'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
>
> Warning: C:\Users\*****\node_modules@pdf-lib\upng\UPNG.js depends on 'pako'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
>
> Recently pako switched with version 2.0.0 to es6. Would it be possible to update the dependency?
Thank you!

### How could this be implemented?

See above

### What problem are you trying to solve?

See above

### Why does this matter to you?

Optimization bailouts, and prevents us from upgrading to Angular 16 (?)

### Would others find this helpful?

Yes

### Are you interested in implementing your proposal?

No

### Why are you submitting a proposal?

To keep the lib alive

**Comment by andreaslarssen** [comment]
Right... and both https://github.com/Hopding/standard-fonts/pull/2 and https://github.com/Hopding/upng/pull/1 are dead in the water...

How about forking the dependencies as well?

**Comment by Sharcoux** [comment]
Ok, let's do that:

- upng is a forked from [photopea ](https://github.com/photopea)/
[UPNG.js](https://github.com/photopea/UPNG.js) which, apparently, is still active. We can try to switch to that lib. Can you open a PR there to include the index.d.ts file from Hopding lib in the project and in the package.json?
- standard-fonts seems dead in both the original and Hopding trees. Can you fork it and try to release it? We can switch to your tree

**Comment by andreaslarssen** [comment]
I'm currently working for a client that uses this package, so I would ask them about spending my time on the upng-package. As for the standard-fonts package, I would not be able to maintain this over time, so it seems like a bad idea for me to fork it. Maybe someone who's more invested in the package could do it?

**Comment by Sharcoux** [comment]
Would your client be able to maintain it?

**Comment by andreaslarssen** [comment]
I really doubt that's going to happen

**Comment by DineshSolanki** [comment]
I just published a standard-fonts package with updated dependency, mind trying it out
https://www.npmjs.com/package/@dineshsolanki/standard-fonts

**Comment by Sharcoux** [comment]
@andreaslarssen @DineshSolanki Can someone open a PR on UPNG.js to upgrade pako, or at least open an issue explaining the problem? This way we don't need to open and maintain yet another fork.

**Comment by DineshSolanki** [comment]
> @andreaslarssen 

I highly doubt that it will be acted upon, but might be worth a try

**Comment by sameer-roofr** [comment]
> [@andreaslarssen](https://github.com/andreaslarssen) [@DineshSolanki](https://github.com/DineshSolanki) Can someone open a PR on UPNG.js to upgrade pako, or at least open an issue explaining the problem? This way we don't need to open and maintain yet another fork.

Will [this PR](https://github.com/photopea/UPNG.js/pull/84) solve it?

**Comment by Sharcoux** [comment]
Probably.

**Comment by Sharcoux** [comment]
The readme now explicitely recommends the override/resolutions. I'll keep this opened in case one day we succeed to fix all deps.
